app/views/casework/assigned_applications/index.html.erb

```diff
@@ -27,19 +27,14 @@
   <div class="govuk-grid-row">
       <div class="govuk-grid-column-full">
         <table class="govuk-table app-dashboard-table">
-          <thead class="govuk-table__head">
-            <tr class="govuk-table__row">
-              <%= table_headers(
-                    %w[
-                      applicant_name
-                      reference
-                      submitted_at
-                      time_passed
-                    ],
-                    search: @search
-                  ) %>
-            </tr>
-          </thead>
+          <%= render DataTable::HeadComponent.new(sorting: @search.sorting) do |head|
+                head.with_row do |row|
+                  row.with_cell(colname: :applicant_name)
+                  row.with_cell(colname: :reference)
+                  row.with_cell(colname: :submitted_at)
+                  row.with_cell(colname: :submitted_at, text: sanitize(thead_text(:time_passed)))
+                end
+              end %>
```

Comment:
Can we aline the %> with the <%= please, like:

 ```ruby
        <%= render DataTable::HeadComponent.new(sorting: @search.sorting) do |head|
            <tr class="govuk-table__row">
                head.with_row do |row|
              <%= table_headers(
                  row.with_cell(colname: :applicant_name)
                    %w[
                  row.with_cell(colname: :reference)
                      applicant_name
                  row.with_cell(colname: :submitted_at)
                      reference
                  row.with_cell(colname: :submitted_at, text: sanitize(thead_text(:time_passed)))
                      submitted_at
                end
                      time_passed
              end
 %>
```

Reply:
use two spaces instead of four per row though - just difficult to show this in github comments 👍

Reply:
erblint forces you to align like this (I'll dig around to see if there is another way.)

Reply:
Can't find another way. erb tabs start from the first ruby character contained within the tag.